sdk/platform/src/hyper/mod.rs

```diff
@@ -16,6 +16,8 @@ pub use hyper::{Method, StatusCode};
 pub mod body;
 
 const CONTENT_TYPE: &str = "content-type";
+const USER_AGENT: &str = "User-Agent";
+const HARBOR: &str = "SBOM-Harbor";
```

Comment:
I don't think a reference to "Harbor" belongs in the platform crate right?

Reply:
Oh, crud.  Yes!

Reply:
I changed this to: `const HYPER_CLIENT: &str = "Hyper-Client";`